src/async/dirwatcher/DirWatcher.js

```diff
@@ -0,0 +1,22 @@
+import {
+  EventEmitter
+} from 'events';
+import chokidar from 'chokidar';
+import {
+  EVENT_TYPE
+} from '../constants';
+
+
+export default class DirWatcher extends EventEmitter {
+
+  watch(path, delay) {
+    const watcher = chokidar
+      .watch(path, {
+        usePolling: true,
+        interval: delay
+      });
+
+    watcher
+      .on("all", () => this.emit(EVENT_TYPE.DIRWATCHER_CHANGED, path));
```

Comment:
`this.emit(EVENT_TYPE.DIRWATCHER_CHANGED, path)`

Every time something changes, event payload will contain `path` to watched directory. It should probably emit path of changed file